### DISCUSSION TITLE
Is there a suggested feed protocol?

### DISCUSSION THREAD

**Discussion by ChristopherJacob** [mrlt8/docker-wyze-bridge#305] [Q&A]
I have switched between RTSP, RTMP, and HLS, but I haven't been able to figure out the pros and cons for each. Does anyone have a doc they can point me to? Or suggestions from your own experience?

**Comment by mrlt8** [comment]
It depends on your use case, but I think most people use the RTSP stream.


From the [rtsp-simple-server readme](https://github.com/aler9/rtsp-simple-server/tree/67e0db13ab3981c9a5059615ffc0365a63f0ca95):

- RTSP is the fastest way to publish and receive streams
- RTMP allows to interact with legacy servers or software (like OBS Studio)
- HLS allows to embed streams into a web page
